lib/vips/methods.rb

```diff
@@ -1,2771 +1,2769 @@
-#   @option opts [Float] :angle1 Output Detected rotation
-#   @option opts [Float] :dy1 Output Detected first-order displacement
-#   @option opts [Float] :dx1 Output Detected first-order displacement
-#   @return [Vips::Image, Hash<Symbol => Object>] Output image, Hash of optional output items
-
-# @!method mosaic1(sec, direction, xr1, yr1, xs1, ys1, xr2, yr2, xs2, ys2, **opts)
-#   First-order mosaic of two images.
-#   @param sec [Vips::Image] Secondary image
-#   @param direction [Vips::Direction] Horizontal or vertcial mosaic
-#   @param xr1 [Integer] Position of first reference tie-point
-#   @param yr1 [Integer] Position of first reference tie-point
-#   @param xs1 [Integer] Position of first secondary tie-point
-#   @param ys1 [Integer] Position of first secondary tie-point
-#   @param xr2 [Integer] Position of second reference tie-point
-#   @param yr2 [Integer] Position of second reference tie-point
-#   @param xs2 [Integer] Position of second secondary tie-point
-#   @param ys2 [Integer] Position of second secondary tie-point
-#   @param opts [Hash] Set of options
-#   @option opts [Integer] :hwindow Half window size
-#   @option opts [Integer] :harea Half area size
-#   @option opts [Boolean] :search Search to improve tie-points
-#   @option opts [Vips::Interpolate] :interpolate Interpolate pixels with this
-#   @option opts [Integer] :mblend Maximum blend size
-#   @option opts [Integer] :bandno Band to search for features on
-#   @return [Vips::Image] Output image
-
-# @!method matrixinvert(**opts)
-#   Invert an matrix.
-#   @param opts [Hash] Set of options
-#   @return [Vips::Image] Output matrix
-
-# @!method match(sec, xr1, yr1, xs1, ys1, xr2, yr2, xs2, ys2, **opts)
-#   First-order match of two images.
-#   @param sec [Vips::Image] Secondary image
-#   @param xr1 [Integer] Position of first reference tie-point
-#   @param yr1 [Integer] Position of first reference tie-point
-#   @param xs1 [Integer] Position of first secondary tie-point
-#   @param ys1 [Integer] Position of first secondary tie-point
-#   @param xr2 [Integer] Position of second reference tie-point
-#   @param yr2 [Integer] Position of second reference tie-point
-#   @param xs2 [Integer] Position of second secondary tie-point
-#   @param ys2 [Integer] Position of second secondary tie-point
-#   @param opts [Hash] Set of options
-#   @option opts [Integer] :hwindow Half window size
-#   @option opts [Integer] :harea Half area size
-#   @option opts [Boolean] :search Search to improve tie-points
-#   @option opts [Vips::Interpolate] :interpolate Interpolate pixels with this
-#   @return [Vips::Image] Output image
-
-# @!method globalbalance(**opts)
-#   Global balance an image mosaic.
-#   @param opts [Hash] Set of options
-#   @option opts [Float] :gamma Image gamma
-#   @option opts [Boolean] :int_output Integer output
-#   @return [Vips::Image] Output image
-
+    # @!method self.system(cmd_format, **opts)
+    #   Run an external command.
```

Comment:
This file is generated automatically, you'll need to change the generator rather than editing it.

Reply:
I see, let's skip it from linting

Reply:
Will skip it from linter.

Reply:
done